### ISSUE TITLE
user not authenticated properly

### ISSUE THREAD

**Issue by vaseem15905** [LiveWiresSRM2023/SRM_CURIOUS_BUG_REACT#10]
when the profile page gets reloaded , couldn't authenticate the user , so ended up with the error "user is not authenticated" .

**Comment by vaseem15905** [comment]
when "auth.onAuthStateChanged()" is used to get , the firebase related data , it is working perfectly . and also we found that the firebase was initialized twice which should be initialized only once . by addressing that , now fixed that too and It it Working perfectly now! :)

**Comment by vaseem15905** [comment]
as we fixed the reload issue , we found another issue on  authenticating the user , like if i logged in with  the email which i already logged in ,  it is redirecting to home page directly  , which is perfectly alright , but only thing is , it is not authenticating the user and fetching the data of the user from firebase .

**Comment by vaseem15905** [comment]
the application is working perfectly when the tab is opened only on guest tab in browser , which is completely wrong approach

**Comment by DEVANSARAVANAN** [comment]
I got fixed this issue. Now, User can log in with the email id which user already used to sign in. Still user should click on sign button to auto sign in, suppose they tried to access any pages directly. I think there is not much bigger issue as of now. we need to verify the person if anyone try to access any pages directly. when we need more effective user experience we should find out other way to do it..
